cmd/zpool/zpool_main.c

```diff
@@ -9075,6 +9075,13 @@ print_history_records(nvlist_t *nvhis, hist_cbdata_t *cb)
 			(void) strftime(tbuf, sizeof (tbuf), "%F.%T", &t);
 		}
 
+		if (nvlist_exists(rec, ZPOOL_HIST_ELAPSED_NS)) {
+			uint64_t elapsed_ns = fnvlist_lookup_int64(records[i],
+			    ZPOOL_HIST_ELAPSED_NS);
+			(void) sprintf(tbuf + strlen(tbuf),
```

Comment:
nit: even though overflow shouldn't be possible how about using `snprintf()` here to keep the various static analysis tools, gcc, clang and even macos all happy.  See #10766, #10400, and others.

Reply:
sure thing